DESCRIPTION

```diff
@@ -20,7 +20,7 @@ Depends:
     rClr (>= 0.9.1)
 Imports: 
     dplyr (>= 1.0.0),
-    ospsuite.utils (>= 1.2.0),
+    ospsuite.utils (>= 1.2.0.9000),
```

Comment:
Why do we need to rely on the .9000 version of the package?

Reply:
Because some functions (e.g. `objectCount()`) I am using are not present in 1.2.0 version.

Reply:
Then I would suggest that we increment the **ospsuite.utils** minor (or major?) version and require 1.3 (or 2.0) as the minimal version.
What do you think?

Reply:
Sure.

I am also happy to make a new CRAN release `1.3.0` today or tomorrow, and we can use that for this PR. 

`1.2.0.9000` is deliberate. 

[R package development idiom](https://r-pkgs.org/release.html#post-release) is `x.x.x` mark CRAN release and then `.xxxx` denote the development version. 

![image](https://user-images.githubusercontent.com/11330453/160785555-0516fe6b-4532-4db2-a4f3-cd081366ccaf.png)

So by relying on that version is making it explicit that I am using a development version of the package and the user will also need to update to that development version if they wish to use _ospsuite_.

Reply:
>I am also happy to make a new CRAN release 1.3.0 today or tomorrow

If the current ospsuite.utils version is stable and will not change soon - why not? :)
Otherwise, we could increment the version to 1.3 (and in this PR) on GitHub first and release on Cran later.
Just let me know soon, because the current Reporting Engine Builds do not run right now, so depending on if you make the Cran release right now or not, RE can either wait for it or must switch from the CRAN version to the latest GitHub-version of the ospsuite.utils

Reply:
Once https://github.com/Open-Systems-Pharmacology/OSPSuite.RUtils/pull/98 is merged, I will make a new CRAN release.